src/Control/Carrier/Class.hs

```diff
@@ -1,8 +1,15 @@
-{-# LANGUAGE DeriveFunctor, EmptyCase, FlexibleInstances, FunctionalDependencies, TypeOperators, UndecidableInstances #-}
+{-# LANGUAGE ConstraintKinds, DeriveFunctor, EmptyCase, FlexibleInstances, FunctionalDependencies, TypeOperators, UndecidableInstances #-}
 module Control.Carrier.Class
 ( Carrier(..)
+, Has
+, send
+  -- * Re-exports
+, (:+:) (..)
+, run
+, module Control.Effect.Class
```

Comment:
NB: we had also re-exported `PureC` from `Control.Carrier`. I don’t _think_ that’s super important for modules defining either effects or carriers, but it is a difference. (Effects merely consuming effects or carriers wouldn’t have been importing this anyway.)